R/tfpscanner.R

```diff
@@ -650,28 +748,37 @@ message(paste('Starting scan', Sys.time()) , '\n')
 
 
 #' Run a fast treedater/mlesky analysis for a given node in the scanner output
-#' @import mlesky
-#' @import treedater
+#' @param u integer
```

Comment:
Is 'u' the ID for a node in the tree?

Reply:
I think so though not entirely sure - have asked Erik. (This is also not a function we'll need to use in the app)